package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -354,13 +360,13 @@ interface BulletProps extends TextProps {
 }
 
 const Bullet = ({ index, style }: BulletProps) => (
-  <Text key={0} style={[style, defaultMarkdownStyles.listItemNumber]}>
+  <Text key={0} style={style}>
     {index ? `${index}. ` : '\u2022 '}
   </Text>
 );
 
-const ListRow = (props: PropsWithChildren<ViewProps>) => (
-  <Text style={[props.style, defaultMarkdownStyles.listRow]}>{props.children}</Text>
```

Comment:
same here.. should not be removed in my view

Reply:
As above 👍